be/src/runtime/task_group/task_group.h

```diff
@@ -0,0 +1,78 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+#pragma once
+#include <queue>
+
+#include "olap/olap_define.h"
+
+namespace doris {
+
+namespace pipeline {
+class PipelineTask;
+}
+
+class QueryFragmentsCtx;
+
+namespace taskgroup {
+
+class TaskGroup;
+
+class TaskGroupEntity {
```

Comment:
What 's the difference between ```TaskGroupEntity```  and ```TaskGroup```?
How about combining both to ```TaskGroup``` ?

Reply:
TaskGroupEntity is a sched entity, maybe a scan task, exec task or task group in the future.